tests/installation/usb_install.pm

```diff
@@ -29,19 +29,43 @@ sub run {
     assert_script_run("set -o pipefail");
 
     # Specify the two usb drives to use
-    my $cmd = "ls /dev/disk/by-id/ -l | grep -i usb | "
-      . "grep -i -v -E \"generic|part|Virtual\" | sed 's#^.*\\\/##'";
-    my @usb = split('\n', script_output($cmd));
+    my $cmd = "ls -l /dev/disk/by-id/ | grep -i usb | grep -i -v -E 'part|Virtual'";
```

Comment:
@nanzhg Hi Nan, will completely removing `generic` filtering be problematic for other SUTs like vh015, vh081, scooter, kermit? I think it will be the case for some machines because usb with `generic` is not usb disk for some machines. Better to run verification on all of them.

Reply:
I had a check on all these SUTs, only vh015 and scooter has usb disk with generic keyword, others do not.

```
scooter: only below 2 disks with usb keyword, and both need to be kept to do 2 usb test
lrwxrwxrwx 1 root root  9 May 28 04:38 usb-Generic_Flash_Disk_A175B6E7-0:0 -> ../../sdb
lrwxrwxrwx 1 root root  9 May 28 04:38 usb-Intenso_Alu_Line-0:0 -> ../../sda


vh015: below disks all have usb keyword, and the first disk with "Generic-_SD" keyword needs to be excluded
lrwxrwxrwx 1 root root  9 Apr 30 03:43 usb-Generic-_SD_MMC_CRW_29203008282014000-0:0 -> ../../sdc
lrwxrwxrwx 1 root root  9 Apr 30 03:43 usb-Kingston_DataTraveler_3.0_E0D55E6B6466F7A0883003AE-0:0 -> ../../sdb
lrwxrwxrwx 1 root root 10 Apr 30 03:43 usb-Kingston_DataTraveler_3.0_E0D55E6B6466F7A0883003AE-0:0-part1 -> ../../sdb1
lrwxrwxrwx 1 root root 10 Apr 30 03:43 usb-Kingston_DataTraveler_3.0_E0D55E6B6466F7A0883003AE-0:0-part2 -> ../../sdb2
lrwxrwxrwx 1 root root  9 Apr 30 03:43 usb-Kingston_DataTraveler_3.0_E0D55E6C382AF7A0889903DC-0:0 -> ../../sdd

Reply:
Probably, the easy way is to make it as a variable. What do you think?
```
my $usb_dev_filter = get_var('USB_DEVICE_FILTER');
my $cmd = "ls -l /dev/disk/by-id/ | grep -i usb | grep -i -v -E '" . $usb_dev_filter . "'";
```

Reply:
Good point! Better to have some default value when no customized value is given, eg what it is now. Performance team , Jerry, was asking me if their usb disk can be recognized and how. But theirs does not even have usb prefix. See below. So, will be good if this PR can survive their case too.

```
ata-Fanxiang_S500PRO_2TB_FXS500PRO230905272
ata-Fanxiang_S500PRO_2TB_FXS500PRO230905272-part1
ata-Fanxiang_S500PRO_2TB_FXS500PRO230905272-part2
ata-Fanxiang_S500PRO_2TB_FXS500PRO230905272-part3
```

Reply:
OK, agree. Will keep some default values for common use. How about to keep the keyword 'part' and 'Virtual' as default?
```
my $usb_disk_filter = get_var('USB_DISK_FILTER');
$usb_disk_filter = '|' . $usb_disk_filter if ($usb_disk_filter);
my $cmd = "ls -l /dev/disk/by-id/ | grep -i usb | grep -i -v -E 'part|Virtual" . $usb_disk_filter . "'";
```

Reply:
This part `grep -i usb` needs to be part of USB_DISK_FILTER too, otherwise performance team's  disk won't be recognized.

Sth like this would work, I think:

```
my $usb_disk_filter = get_var('USB_DISK_FILTER')? get_var('USB_DISK_FILTER') : "grep -i usb | grep -i -v -E 'part|Virtual'";
my $cmd = "ls -l /dev/disk/by-id/ | " . $usb_disk_filter;
```

Reply:
@alice-suse, I've updated code as per your suggestion, and the verification has got this part passed.